### ISSUE TITLE
Create backend Schema for Likes 

### ISSUE THREAD

**Issue by adarshanand67** [adarshanand67/ShareMe#7]

**Comment by adarshanand67** [comment]
> Hello @adarshanand67, I would like to work on this issue under JWOC 2023.

Already assigned issue no #13 kindly work on it before this

**Comment by Laxmankohar** [comment]
@adarshanand67 , I would like to work on this issue under JWOC. Can you please assign it to me?

**Comment by Laxmankohar** [comment]
@adarshanand67 bro, please check it and if need some modification, please tell me.

![image](https://user-images.githubusercontent.com/73365025/221962178-871c3030-bd8a-4688-9670-b4a4ad4141b7.png)

**Comment by adarshanand67** [comment]
Yup you are going correct , basically storing all those people who liked the post in an object.

**Comment by Laxmankohar** [comment]
Should i make PR?

**Comment by adarshanand67** [comment]
Can you also test it and show me the results as SS

**Comment by Laxmankohar** [comment]
Bro , i am not able to login in sanity.io

![image](https://user-images.githubusercontent.com/73365025/222057515-395b1a9c-5950-4c1c-9b74-6033bccb9eca.png)

**Comment by Laxmankohar** [comment]
May be you have to add me as a collaborator

**Comment by Laxmankohar** [comment]
Bro, Should i make PR?

**Comment by Laxmankohar** [comment]
> Bro, Should i make PR?

.
